target/linux/ipq40xx/image/Makefile

```diff
@@ -174,11 +182,18 @@ define Device/engenius_ens620ext
         BLOCKSIZE := 64k
```

Comment:
I think some tabs vs whitespaces slipped by. This is not your problem though. I'll fix this when I add your patch.

Reply:
Sorry, thats a left-over from the original commit. Since I will be fixing things from the checkpatch.pl I'll pick that up too.

What is your preference (or best practice): squash commits, or a separate commit with the requested fixes?